samples/app/src/main/kotlin/com/bumble/appyx/app/node/samples/SamplesContainerNode.kt

```diff
@@ -128,18 +146,110 @@ class SamplesContainerNode(
 
     @Composable
     private fun SamplesSelector(backStack: BackStack<NavTarget>) {
-        Column(
-            horizontalAlignment = Alignment.CenterHorizontally
+        val decorator: @Composable (child: ChildRenderer) -> Unit = remember {
+            {
+                ScaledLayout() {
+                    it.invoke()
+                }
+
+            }
+        }
+        LazyColumn(
+            modifier = Modifier.fillMaxSize(),
+            contentPadding = PaddingValues(8.dp),
+            verticalArrangement = Arrangement.spacedBy(8.dp, Alignment.Top)
+
         ) {
-            Button(onClick = { backStack.replace(NavTarget.OnboardingScreen) }) {
-                Text("What is Appyx?")
+            item {
+                SampleItem(
+                    title = "Dating cards NavModel",
+                    subtitle = "Swipe right on NavModel concept",
+                    onClick = { backStack.push(NavTarget.CardsExample) },
+                ) { PermanentChild(navTarget = NavTarget.CardsExample, decorator = decorator) }
             }
-            Button(onClick = { backStack.push(NavTarget.CardsExample) }) {
-                Text("Dating cards NavModel")
+            item {
+                SampleItem(
+                    title = "What is Appyx?",
+                    subtitle = "Launch Appyx onboarding flow to explore main ideas in a set of slides",
+                    modifier = Modifier
```

Comment:
Do we need it? `SampleItem` internally uses modifier.fillMaxSize().aspectRatio(16f / 9)